jwst/regtest/test_miri_lrs_slit_spec2.py

```diff
@@ -25,6 +25,9 @@ def run_pipeline(rtdata_module):
             "--steps.srctype.save_results=true",
             "--steps.flat_field.save_results=true",
             "--steps.pathloss.save_results=true",
+            "--steps.photom.save_results=true",
```

Comment:
There's no need to save and compare the photom step results directly, because they should be identical to the "cal" product (because photom is the last step before saving the cal file).